order.go

```diff
@@ -64,10 +64,16 @@ func (c *Client) CreateOrderWithPaypalRequestID(ctx context.Context,
 
 // UpdateOrder updates the order by ID
 // Endpoint: PATCH /v2/checkout/orders/ID
-func (c *Client) UpdateOrder(ctx context.Context, orderID string, purchaseUnits []PurchaseUnitRequest) (*Order, error) {
+func (c *Client) UpdateOrder(ctx context.Context, orderID string, op string, path string, value map[string]string) (*Order, error) {
```

Comment:
Seems like this API hasn't worked at all given the documentation here: https://developer.paypal.com/api/orders/v2/#orders_patch

@plutov I think it makes sense to make this change even though it is a breaking change. Curious what you think